docs/docs/k8s/ingress.md

```diff
@@ -124,6 +124,18 @@ metadata:
 
 :::
 
+::: tip
+Routes are sorted and applied in the following order:
+
+- ascending by `from`,
+- descending by `path`,
+- descending by `regex`,
+- descending by `prefix`,
+- ascending by `id`.
+
```

Comment:
Overall LGTM. Some minor notes:

- It took me a bit of time to understand in context why this section exists here when looking at the preview.
- I don't understand why you arranged it as Ascending, then 3 Descending, then Ascending again. I don't feel strongly about it but perhaps put both Ascending together? If there's a specific reason disregard.

Reply:
@cmo-pomerium that's the order in which they are processed. Perhaps I should make it a numbered list, so it's more clear that it's ordered?